casepro/backend/junebug.py

```diff
@@ -40,6 +46,74 @@ def get_addresses(self, uuid):
         return (
             a['address'] for a in addresses if a.get('address') is not None)
 
+    def get_identities(self, **kwargs):
+        '''Get the list of identities filtered by the given kwargs.'''
+        url = '%s/api/v1/identities/search/?' % (self.base_url)
+        params = {}
+        for key, value in kwargs.iteritems():
+            params[key] = value
+
+        identities = self.get_paginated_response(
+            url, params=params)
+
+        return (
+            IdentityStoreContact(i) for i in identities if
+            i.get('details').get('name') is not "removed"
+        )
+
+
+class IdentityStoreContact(object):
+    """
+    Holds identity data for syncing
+    """
+    def __init__(self, json_data):
+        for k, v in json_data.items():
+            setattr(self, k, v)
+
+        # Languages in the idenity store have the country code at the end
+        remote_language = json_data.get('details').get('preferred_langage')
+        language = remote_language[:remote_language.index('_')]
```

Comment:
It's probably safer to do `_, _, language = remote_language.rpartition('_')`. It's a bit arcane but guards against edge cases where the string doesn't contain an `_`.